### PR TITLE
Add Actions 

### PR DESCRIPTION
**Pull request by sbtoonz**
for testing PR's/Commits as well as building on push to branch named prod

this action directly references the csproj for ValheimExportHelper for the Version tag and will use that # to post releases upon seeing a push to "prod"

Signed-off-by: zarboz <sbtoonz@gmail.com>

### CODE REVIEW

File: .github/workflows/ProductionRelease.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      run: dotnet restore
+    - name: Build
+      run: dotnet build --no-restore
+    - name: Test Build
+      run: dotnet test --no-build --verbosity normal
+    - name: Setup build artifacts
+      run: |
+        mkdir Artifacts
+        mv ValheimExportHelper/bin/Debug/net6.0/ValheimExportHelper.dll  Artifacts/ValheimExportHelper.dll
+        mv ValheimExportHelper/bin/Debug/net6.0/YamlDotNet.dll  Artifacts/YamlDotNet.dll
```
Comment (heinermann) [inline .github/workflows/ProductionRelease.yml]: We should build Release and not Debug right?

File: .github/workflows/ProductionRelease.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        cat ValheimExportHelper/ValheimExportHelper.csproj | grep '<Version>' > tmp.txt
+        sed 's|[<Version>,]||g' tmp.txt | sed 's|[/,]||g' > VERSION
+      
+    - name: Commit Version
+      uses: EndBug/add-and-commit@v7
+      with:
+        author_name: "Bryan Smith"
+        author_email: "sbtoonz@gmail.com"
+        message: "Update Version"
+        add: "VERSION"
```
Comment (heinermann) [inline .github/workflows/ProductionRelease.yml]: What does this do? (or rather what's its purpose)

Comment (sbtoonz) [reply .github/workflows/ProductionRelease.yml]: > What does this do? (or rather what's its purpose)

It should be redone to a generic author but it commits the Version tag from the SLN -> VERSION file to extract for the Tag/Name

File: .github/workflows/ProductionRelease.yml (hunk 1 above)
Comment (sbtoonz) [reply .github/workflows/ProductionRelease.yml]: Whoops! good catch

File: .github/workflows/ProductionRelease.yml (hunk 2 above)
Comment (heinermann) [reply .github/workflows/ProductionRelease.yml]: Is it not obtainable without a commit?

Comment (sbtoonz) [reply .github/workflows/ProductionRelease.yml]: Resolved

**Pull request merged**